src/setup/rgw_setup

```diff
@@ -34,6 +37,22 @@ class SetupCmdBase(Cmd):
         parser.add_argument('--services', default='services', help='services')
         parser.add_argument('--config', default='config_url', help='config')
 
+    def get_cortx_config(self, cortx_config_url: str):
+        """Load CORTX GConf."""
+        self.cortx_gconf = MappedConf(cortx_config_url)
+        return self.cortx_gconf
+
+    def initialize_logging(self, cortx_gconf: MappedConf):
+        """Initialize Logging."""
+        log_path = cortx_gconf.get('cortx>common>storage>log')
+        if log_path is None:
+            raise RgwSetupError(errno.EINVAL, 'Log path is None.')
+        rgw_log_path = os.path.join(log_path, 'rgw', Conf.machine_id)
+        if not os.path.exists(rgw_log_path):
```

Comment:
No need of this check. os.makedirs() handles it already.

Reply:
ACK.